### PR TITLE
Add comments that 'unsafe-eval' needed for vue.js to work in application.rb

### PR DESCRIPTION
**Pull request by mytram**
Documentation improvement (#729) Thanks, -Mytram

### CODE REVIEW

File: lib/hanami/generators/app/application.rb.tt [hunk 1]
```diff
@@ -205,6 +205,12 @@ module <%= config[:classified_app_name] %>
       # Inline JavaScript is NOT allowed. To enable it, please use:
       # "script-src 'unsafe-inline'".
       #
+      # Javascript eval() and similar methods are NOT allowed. It needs to be enabled
+      # for Javascript frameworks that rely on
+      # such features, such as vue.js. To enable it, please use:
+      # "script-src unsafe-eval'".
```
Comment (cllns) [inline lib/hanami/generators/app/application.rb.tt]: What do you think about wording this as:
```
      # Inline JavaScript is NOT allowed. To enable it, please use:
      # "script-src 'unsafe-inline'".
      #
      # If you need to use JavaScript's eval() and similar functions, 
      # which some frameworks require, please use:
      # "script-src unsafe-eval'".
```
?

I'd prefer to stay away from mentioned Vue.js specifically. The place to handle that is in a guide for Vue.js + Hanami (which is planned).

Comment (davydovanton) [inline lib/hanami/generators/app/application.rb.tt]: @cllns what do you think about this:

```
# Some js frameworks (Vue, react, etc) use inline JavaScript
# And hanami don't allow it bu default. To enable it, please use:
# "script-src 'unsafe-inline'".
```

Comment (cllns) [inline lib/hanami/generators/app/application.rb.tt]: I don't know much about how JS MVC frameworks work in the browser. But there's `unsafe-inline` and `unsave-eval`. 

Vue needs the latter (according to @mytram), but I'm not sure about the others :)

**Comment by davydovanton** [comment]
@mytram thanks for PR! 💛 
Could you fix @cllns issue and update tests?

File: lib/hanami/generators/app/application.rb.tt (hunk 1 above)
Comment (jodosha) [inline lib/hanami/generators/app/application.rb.tt]: I'd avoid a direct mention to Vue.js as well.

**Comment by jodosha** [comment]
I've just noticed that we already have documented this issue here: https://github.com/hanami/hanami/blob/master/lib/hanami/generators/app/application.rb.tt#L205-L206

```ruby
      # Inline JavaScript is NOT allowed. To enable it, please use:
      # "script-src 'unsafe-inline'".
```

The changes of this PR will be redundant. I'm closing this. Thank you @mytram

**Pull request closed without merging**